clusters/app.ci/dex/manifests.yaml

```diff
@@ -167,6 +172,16 @@ spec:
             secretKeyRef:
               name: dpcr-secret
               key: dpcr-id
+        - name: hive-SECRET
```

Comment:
This doesn't match the above env var either

/lgtm cancel

Reply:
updated.